### DISCUSSION TITLE
[A-L] (2023/24) Foglio 12 - Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#269] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-11-28 131710](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/8aec2253-99cb-4a09-a2fd-13f5565b607d)

**Comment by daromacarlo** [comment]
![Immagine 2023-11-30 171740](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/8412704b-2459-4e2b-99fb-2868f86c8c00)
![Immagine 2023-11-30 171754](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/b0391b33-9ec7-4648-9052-df635bf5b236)
![Immagine 2023-11-30 171809](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/d688120f-c573-44dd-8321-4f81b7e22e80)
![Immagine 2023-11-30 171823](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/96c34532-2eba-4e92-b8f7-41945434844b)

**Comment by CiottoloMaggico** [comment]
@CarloDaRomadev ancora non so cos'è il rango ma penso sia giusta
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/4405c332-fe6d-40d9-8830-d2ebf4ff27cc)

**Comment by daromacarlo** [comment]
![Immagine 2024-01-25 161457](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/4b69b26e-a507-4ad6-8554-a65a6a438574)
![Immagine 2024-01-25 161517](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/cecd60af-e214-4814-96d5-cb0e5ef989fa)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/cef0d5c4-7849-4441-a152-6de92b76c094)

![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/61bc4a5d-e7f2-4c71-8b16-8ad3c24d4aec)
